src/components/SearchBar/SearchBar.module.scss

```diff
@@ -6,32 +6,54 @@
 .searchBar input {
 	width: 100%;
 	height: 32px;
-	background: var(--color-cosmic-gradient-25);
+	background: none;
+	border: 1px solid var(--color-purple-lighter);
 	border-radius: 31px;
 	padding: 0 24px 1px 39px;
 
 	/* Text */
 	text-align: left;
-	color: white;
+	color: #bfbfbf;
 
 	/* Text shadow isn't working as expected - it cuts off for some reason */
 	/* text-shadow: var(--glow-text-white); */
 }
 
-.searchBar button {
+.searchBar .ButtonIconStyles {
 	position: absolute;
 	padding: 0;
 	left: 16px;
 	top: 9px;
 	height: 14px;
 	width: 14px;
+	background-image: url('./assets/search-icon-gray.svg');
 	background-color: rgba(0, 0, 0, 0);
-	background-image: url('./assets/search-icon.svg');
 	background-position: center;
 	background-size: 14px 14px;
 	background-repeat: no-repeat;
 }
 
+.searchBar:focus-within .ButtonIconStyles {
+	background-image: url('./assets/search-icon.svg');
```

Comment:
There is probably a nicer way to do this with CSS brightness or similar, instead of changing the asset. Not a major issue though

Reply:
Cool. I've applied a filter instead of changing the asset. It's been transitioned too.